addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -86,8 +86,13 @@ def action_invite(self):
 
     def _prepare_mail_values(self, slide_channel_partner):
         """ Create mail specific for recipient """
-        subject = self._render_field('subject', slide_channel_partner.ids)[slide_channel_partner.id]
-        body = self._render_field('body', slide_channel_partner.ids)[slide_channel_partner.id]
+        set_lang, lang_opt = {}, {}
+        subject_same_as_template = (self.template_id and self.subject == self.template_id.subject)
+        if self.body_has_template_value and subject_same_as_template:
```

Comment:
Note that we could split (don't have to have both identical to check for translations, each one is separate)

Reply:
Hello :)

> Indeed we should check the returned result directly, here it is a bit indirect. Note that you can create a small template and its translation in the test, so that we don't rely on existing data.

I wanted to do something like this, but I'm not sure how I would achieve this.
As far as I can tell, the translations are stored in the db as jsonb, and the orm does not really provide a way to write to them, so we would have to do some raw SQL queries in the test? (I guess I'm missing something)

> Note that we could split (don't have to have both identical to check for translations, each one is separate)

Do you mean that if (for example) the subject is unchanged but the body is, we still want the subject to be translated, I just assumed this was not what we wanted, but I guess it kinda makes sense/would simplify the code. (I think then we could completely remove the checks)

> Side note: invite tests are inside 'test_attendee.py' test file (see notably 'test_invite_to_course'), so would add the test in that file to keep them in the same file :)

Will do :)

Reply:
> I wanted to do something like this, but I'm not sure how I would achieve this.

Just copy/paste the french text from the email. No code.

Reply:
But then you rely on data ... and actual translation (you could have modified it in your db). Would say to create a small template and its translation so that the test is deterministic. To update it in new version, just update its body in a given environment should do the trick.

Reply:
@tde-banana-odoo I wanted to r+ this one but I see that this question has been left open:

>> Note that we could split (don't have to have both identical to check for translations, each one is separate)

> Do you mean that if (for example) the subject is unchanged but the body is, we still want the subject to be translated? I just assumed this was not what we wanted, but I guess it kinda makes sense/would simplify the code. (I think then we could completely remove the checks)

Reply:
Yes, in the end what we want is to try to use translations if possible :)

Reply:
@tde-banana-odoo Hello :)
I made the change